pkg/anago/anago.go

```diff
@@ -57,6 +58,9 @@ type Options struct {
 	// The build version to be released. Has to be specified in the format:
 	// `vX.Y.Z-[alpha|beta|rc].N.C+SHA`
 	BuildVersion string
+
+	// The semver variant of the BuildVersion
+	semVer semver.Version
```

Comment:
I have a similar change in https://github.com/kubernetes/release/pull/1707/files#diff-a2acdd644b079590787e67f51ed616be39d49489dbcaf3582991346b92f47156R62-R64

Let's move it over to a different struct as discussed in the review.